js/src/ValidateSPV.js

```diff
@@ -250,3 +250,126 @@ export function validateHeaderChain(headers) {
   }
   return totalDifficulty;
 }
+
+/**
+ *
+ * Checks validity of an entire bitcoin header
+ *
+ * @dev                   Checks that each element in a bitcoin header is valid
+ * @param {Uint8Array}    header The bitcoin header
+ * @param {Uint8Array}    hash The hash of the header
+ * @param {Uint8Array}    hashLE The LE hash of the header
+ * @param {Number}        height The height
+ * @param {Uint8Array}    merkleRoot The merkle root of the header
+ * @param {Uint8Array}    merkleRootLE The LE merkle root
+ * @param {Uint8Array}    prevHash The hash of the previous header
+ * @returns {Boolean}     True if all the elements of the header are valid
+ * @throws {Error}        If any of the bitcoin header elements are invalid
+*/
+export function validateHeader(
```

Comment:
can we make this accept the deserialized Header object rather than all of its attributes?

Reply:
Done